samples/azure/functions/service-bus/ASBFunctions_0.1/AzureFunctions.ASBTrigger/AzureServiceBusTriggerFunction.cs

```diff
@@ -14,7 +14,7 @@ public class AzureServiceBusTriggerFunction
 
     [FunctionName(EndpointName)]
     public static async Task Run(
-        [ServiceBusTrigger(queueName: EndpointName, Connection = ConnectionStringName )]
```

Comment:
I thought we kept the connection string here because it's non-obvious what the ASB default connection string name is (compared to ASQ where it's already set when setting up a functions account). No strong opinion though, but if we rely on the default value for the sample, shouldn't we remove the `ConnectionStringName`  constant as well or is it used somewhere?

Reply:
We kept it because of the bug, but since it's fixed now, we can keep the sample simpler and coherent with the ASQ. I'll remove the constant. Thank you for catching this, @timbussmann.